test/app.locals.js

```diff
@@ -19,8 +20,8 @@ describe('app', function(){
       var app = express();
       app.set('title', 'House of Manny');
       var obj = app.locals.settings;
-      obj.should.have.property('env', 'test');
-      obj.should.have.property('title', 'House of Manny');
+      assert.strictEqual(obj.env, 'test');
+      assert.strictEqual(obj.title, 'House of Manny');
```

Comment:
These lines didn't work with `should`, so I converted them to use `assert`. For consistency, I converted the whole file.

Happy to open a PR to convert the rest of the project, if that's useful.

Reply:
Makes sense! The `should` lib was on the way out, as it altered global prototypes and generally polluted the testing environment. The `master` branch now has it completed removed, so that'll end up in 5.x soon enough. I def. won't miss `should` :)